### PR TITLE
Sr loomdev

### PR DESCRIPTION
**Pull request by sidraman14**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 1.2.1
+## 1.2.2
```
Comment (Octember) [inline CHANGELOG.md:3]: This is wrong, you'll want a new section for 1.2.2 - this changes the meaning of what `1.2.1` was

**Review by brendonjohn — APPROVED** [review]
lgtm;

File: src/examples/components/GettingStarted.js [hunk 2]
```diff
@@ -17,6 +17,23 @@ class GettingStarted extends Component {
         <InfoBlock title="npm">
           <CodeBlock script="npm install @loomhq/loom-embed" />
         </InfoBlock>
+        <InfoBlock title="script tag">
+          <CodeBlock script={`<script type="module">
+  import * as loom from "https://www.unpkg.com/@loomhq/loom-embed@${packageJson.version}/dist/esm/index.js?module";
```
[the remark this answers is not in the document]
Comment (sidraman14) [reply src/examples/components/GettingStarted.js:22]: This was what was happening on my end (also why I changed it)
https://www.loom.com/share/43ad342ee249411db914fda5b2e6ab7c

[the remark this answers is not in the document]
Comment (brendonjohn) [reply src/examples/components/GettingStarted.js:22]: Interwebsting, this is what I found:

- Works on Firefox
- Fails on Chrome

I spotted that on CodeSandbox they're serving static assets via a ServiceWorker.

Hypothesis: The service worker is broken

## Method for testing:

1. I exported the app from the code sandbox

image

2. extracted the zip and ran yarn inside the directory
3. ran `yarn start` to start the local server running
4. Opened the site in chrome

Looks like it's working on Chrome: 
image


## Summary

Thanks for pushing back on me. Let's leave this as is.

**Pull request merged**